src/main/drivers/max7456.c

```diff
@@ -367,6 +371,22 @@ void max7456ReInit(void) {
     __spiBusTransactionBegin(busdev);
     // Make sure the Max7456 is enabled
     max7456Send(MAX7456ADD_VM0, videoSignalReg);
+
+    if (blackBackground) {
+        /* VM1 takes 8 bits:
+         *
+         * 7 - Background Mode (0- transparent, 1-gray)
+         * 6,5,4 - Background Mode Brightness (000 - black, 111 49% gray)
+         * 3,2 - Blinking time ( 01 is default)
+         * 1,0 - Blinking duty cycle ( 11 is default)
+         */
+        // 135 = 10000111
+        max7456Send(MAX7456ADD_VM1, 135);
```

Comment:
agreed, i'll update it.

Although instead of using a hex, I'd like to find a way to lay out the whole bit sequence bit better in order to make it super clear what's going on.

Reply:
You could change it to hex, then make it a define. Then it's named will make it make more sense.